### DISCUSSION TITLE
AI Tech is moving very fast. Stability released Stable Cascade

### DISCUSSION THREAD

**Discussion by AshD** [saddam213/OnnxStack#116] [General]
Very impressive for a research model. It follows the prompt much better and is very good at text and uses lower VRAM. 
https://www.reddit.com/r/StableDiffusion/comments/1aqgvsc/impressed_with_stable_cascade_in_following/

I added a request for them to release an Onnx format of the model.
https://huggingface.co/stabilityai/stable-cascade/discussions/16

**Comment by saddam213** [comment]
Considering all the huge brand names that are supposed to be supporting ONNX, it blows my mind none of them have bothered to hire a single person to convert models to ONNX format


I would literally pay someone $100NZD to convert a SDV model to onnx for me, this is how ridiculous its getting

**Comment by AshD** [comment]
Try this
https://accelerator.github.com/

I will try to find someone at Microsoft that can help.
